WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.kt

```diff
@@ -431,6 +431,7 @@ class EditPostActivity : BaseAppCompatActivity(), EditorFragmentActivity, Editor
     @Inject lateinit var editorJetpackSocialViewModel: EditorJetpackSocialViewModel
     private lateinit var editPostNavigationViewModel: EditPostNavigationViewModel
     private lateinit var editPostSettingsViewModel: EditPostSettingsViewModel
+    @Inject lateinit var editPostAuthViewModel: EditPostAuthViewModel
```

Comment:
@oguzkocer I resolved a merge conflict with the recent`@Inject`s removal from `editPostNavigationViewModel` in https://github.com/wordpress-mobile/WordPress-Android/pull/22068. I believe retaining the `@Inject` for `editPostAuthViewModel` is the correct, but I request your input on that.